src/vex.dialog.js

```diff
@@ -70,12 +73,32 @@ var plugin = function plugin (vex) {
         options.message = vex._escapeHtml(options.message)
       }
 
+      var defaultYES = 'OK';
+      var defaultNO = 'Cancel';
+
+
+
+     // options.buttons.YES.text = yestxt;
+
+      
       // Build the form from the options
       var form = options.unsafeContent = buildDialogForm(options)
 
       // Open the dialog
       var dialogInstance = vex.open(options)
 
+      var yestxt = options.yesText;
+      if(yestxt !== '')
```

Comment:
We can avoid using an extra variable by always referencing `options.yesText`, instead of using `yestxt`.